contracts/ERC721A.sol

```diff
@@ -88,11 +92,11 @@ contract ERC721A is Context, ERC165, IERC721A {
     /**
      * @dev See {IERC165-supportsInterface}.
      */
-    function supportsInterface(bytes4 interfaceId) public view virtual override(ERC165, IERC165) returns (bool) {
+    function supportsInterface(bytes4 interfaceId) public view virtual override returns (bool) {
```

Comment:
I would leave a comment in the code stating what these values are and where they're coming from

Reply:
Ok, i’ll leave a comment with the EIP165 link.